react/AddToCartButton.tsx

```diff
@@ -109,6 +110,7 @@ function AddToCartButton(props: Props) {
     multipleAvailableSKUs,
     customPixelEventId,
     addToCartFeedback,
+    onClickEventPropagation,
```

Comment:
Since this component is also an entry point we should add the default here too

```suggestion
    onClickEventPropagation = 'disabled',
```